packages/genesys-spark-components/src/components/beta/gux-tooltip-base/gux-tooltip-base.tsx

```diff
@@ -157,8 +164,10 @@ export class GuxTooltipBase {
 
   private setForElement(): void {
     if (this.forElement) {
-      const tooltipId = this.tooltipId ? this.tooltipId : this.id;
-      this.forElement.setAttribute('aria-describedby', tooltipId);
+      if (!this.visualOnly) {
+        const tooltipId = this.tooltipId ? this.tooltipId : this.id;
+        this.forElement.setAttribute('aria-describedby', tooltipId);
+      }
```

Comment:
I tested that one too and it seems to override aria-hidden so things were reading out twice

Reply:
Sorry, I wrote that in a confusing way. I meant you could add `if (!this.visualOnly || Boolean(this.getAttribute('aria-hidden'))` and then it would carry through.